examples/example-migration-configuration/src/main/resources/migrations.json

```diff
@@ -1,11 +1,19 @@
 [
   {
```

Comment:
I: removed Toml stuff since that doesn't really help anyone.  Updates example projects to be simple instantiations of things users might really want to know how to do instead